extensions/amp-ad-exit/0.1/amp-ad-exit.js

```diff
@@ -132,17 +150,27 @@ export class AmpAdExit extends AMP.BaseElement {
          this example will return "medium".
         */
         substitutionFunctions[customVarName] = () => {
```

Comment:
Security warning: `customVarName` must be regex escaped. Since this is already in, please file an issue to fix it (soon) in a followup PR.

Reply:
Created https://github.com/ampproject/amphtml/issues/11617